assets/styles/style.css

```diff
@@ -130,6 +138,42 @@ section {
   padding: 0 7.5%;
 }
 
+.navbar .main-nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+
+.navbar .main-nav .main-menu {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  list-style-type: none;
+}
+
+.navbar .main-nav .main-menu .nav-item .nav-link {
+  padding: 10px;
+  text-decoration: none;
+  color: #344563;
+  font-size: 18px;
+  font-weight: 600;
+  transition: 0.3s ease-out;
+}
+
+.navbar .main-nav .main-menu .nav-item .nav-link:hover {
+  color: #36b37f;
+}
```

Comment:
- [ ] Kindly make sure your designs look exactly like the provided Figma design.

  - ### YOUR DESIGN
     ![image](https://user-images.githubusercontent.com/87197412/201723508-8f4d5352-1da4-4500-a9dd-a1fb19bd0166.png)

  - ### FIGMA DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/201723410-886e9171-1ad0-44c8-b7f3-2c440da7cc8c.png)